f2/src/pdf/PdfDocument.js

```diff
@@ -0,0 +1,113 @@
+import React from 'react'
+import { Page, Text, Link, View, Document, Image } from '@react-pdf/renderer'
+import { pdfStyles } from './pdfStyles'
+import { testdata } from '../ConfirmationSummary'
+import topBannerEn from '../images/topbanner-en.png'
+import topBannerFr from '../images/topbanner-fr.png'
+import footerCanada from '../images/footerCanada.png'
+import betaEn from '../images/beta-en.png'
+import betaFr from '../images/beta-fr.png'
+import langEn from '../locales/en.json'
+import langFr from '../locales/fr.json'
+import { WhoAreYouReportForView } from './WhoAreYouReportForView'
+import { AnonymousView } from './AnonymousView'
+import { ContactInfoView } from './ContactInfoView'
+import { HowDidItStartView } from './HowDidItStartView'
+import { WhenDidItHappenView } from './WhenDidItHappenView'
+import { WhatWasAffectedView } from './WhatWasAffectedView'
+import { MoneyLostInfoView } from './MoneyLostInfoView'
+import { InformationView } from './InformationView'
+import { DevicesView } from './DevicesView'
+import { BusinessInfoView } from './BusinessInfoView'
+import { WhatHappenedView } from './WhatHappenedView'
+import { SuspectCluesView } from './SuspectCluesView'
+import { EvidenceInfoView } from './EvidenceInfoView'
+import { LocationInfoView } from './LocationInfoView'
+
+export const PdfDocument = (props) => {
+  const lang = props.locale === 'fr' ? langFr : langEn
+
+  const impact = {
+    affectedOptions: [],
+    ...testdata.formData.whatWasAffected,
+    ...props.data.formData.whatWasAffected,
+  }
+  const anonymous = {
+    ...testdata.formData.anonymous,
+    ...props.data.formData.anonymous,
+  }
+  const { fyiForm } = props.data.formData
+
+  return (
+    <Document>
+      <Page size="A4" style={pdfStyles.page}>
+        <View style={pdfStyles.header} fixed>
+          <View style={pdfStyles.rowContainer}>
+            <Image
+              style={pdfStyles.topbanner}
+              src={props.locale === 'en' ? topBannerEn : topBannerFr}
+            />
+            <Image
+              style={pdfStyles.beta}
+              src={props.locale === 'en' ? betaEn : betaFr}
+            />
+            <Text style={pdfStyles.betaText}>{lang['pdf.betaText']}</Text>
```

Comment:
Can't we use the Lingui Trans macro here? `<Trans id="pdf.betaText" />`? This would remove the need to have mark ids for lingui section.

Reply:
That's a very good questions. I tried <Trans id="pdf.betaText" /> first, but it doesn't work with react-pdf, can't figure out why. If you figure it out, definitely let me know. Thanks!